src/test/java/seedu/address/model/appointment/UniqueAppointmentListTest.java

```diff
@@ -0,0 +1,170 @@
+package seedu.address.model.appointment;
```

Comment:
Changed implementation of UniqueAppointmentList to UniqueList (generic interface)